### PR TITLE
Some refactoring

### PR DESCRIPTION
**Pull request by YacineSteeve**
## What does this PR do ?

Mainly: 
* Add a target in `Makefile` to seed data
* Make the browser tab icon adapt to the user theme

### CODE REVIEW

**Comment by YacineSteeve** [comment]
Current dependencies on/for this PR:
* main
  * **PR #81** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/81" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/81?utm_source=stack-comment).

File: frontend/public/icon-white.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
+    <path d="M85.518 39.482c-0.976-0.977-2.56-0.977-3.535 0L50 71.464 34.268 55.732c-0.976-0.977-2.56-0.977-3.535 0l-8.75 8.75c-0.977 0.976-0.977 2.559 0 3.535l26.25 26.25C48.72 94.756 49.36 95 50 95s1.28-0.244 1.768-0.732l42.5-42.5c0.977-0.976 0.977-2.559 0-3.535L85.518 39.482zM50 88.964L27.286 66.25l5.214-5.214 15.732 15.732c0.976 0.977 2.56 0.977 3.535 0L83.75 44.786 88.964 50 50 88.964z" fill="#ffffff" data-fill-palette-color="accent"></path>
+</svg>
```
Comment (chrichri17) [inline frontend/public/icon-white.svg:4]: Nit: file should end with a beeline character

**Review by chrichri17 — APPROVED** [review]
Nice 💯

File: frontend/public/icon-white.svg (hunk 1 above)
Comment (YacineSteeve) [reply frontend/public/icon-white.svg:4]: what is it ?

**Pull request merged**